include/tallies/general_tally.hpp

```diff
@@ -25,6 +25,11 @@ class GeneralTally : public ITally {
 
   void score_source(const BankedParticle& p) override final;
 
+  virtual double evaluate(const Position r,
+                          const double E) const override final;
+  virtual std::vector<double> evaluate(const std::vector<Position> r,
+                                       const double E) const override final;
```

Comment:
```suggestion
  virtual std::vector<double> evaluate(const std::vector<std::pair<Position,double>>& r_E) const override final;
```
We should have something like this, so that we could have a different energy with each position.